test/index_test.js

```diff
@@ -242,6 +251,16 @@ describe('Client', () => {
 
         transactions.should.be.an.Array().and.empty();
       });
+
+      it('should support named parameters', async () => {
```

Comment:
Is there a more generic test we can add that doesn't require adding a `should support named parameters` type of test for each and every method?

Reply:
Since each method has its own named arguments, we cannot add a generic test (or at least without adding all method definitions).